### PR TITLE
2.3.2 release

### PR DESCRIPTION
**Pull request by daschuer**
Ready?

### CODE REVIEW

**Comment by ronso0** [comment]
It's 2.3.2 ;)
I changed the PR titlejust now, please reword affected commits.

**Comment by ronso0** [comment]
I  think #4544 and #4531 should be added to the Changelog. I'm preparing the review sugestion.
These are only my PRs I remember from the top of my head, let's double-check.

**Comment by daschuer** [comment]
Ups ... I will reopen another PR at the correct branch

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 * Preserve file creation time when writing metadata on Windows [#4586](https://github.com/mixxxdj/mixxx/pull/4586) [lp1955314](https://bugs.launchpad.net/mixxx/+bug/1955314)
 * Fix handling of file extension when importing and exporting sampler settings [#4539](https://github.com/mixxxdj/mixxx/pull/4539)
 * Fix crash when using an empty directory as resource path using the `--resource-path` command line option [#4575](https://github.com/mixxxdj/mixxx/pull/4575) [lp1934560](https://bugs.launchpad.net/mixxx/+bug/1934560)
+* Added mapping for Pioneer DDJ SB3 controller [#3821](https://github.com/mixxxdj/mixxx/pull/3821)
```
Comment (ronso0) [inline CHANGELOG.md:11]:
```suggestion
* Added mapping for Pioneer DDJ SB3 controller [#3821](https://github.com/mixxxdj/mixxx/pull/3821)
* Don't wipe sound config during startup if configured devices are unavailable [#4544](https://github.com/mixxxdj/mixxx/pull/4544)
* Append selected file extension when exporting to plalist files [#4531](https://github.com/mixxxdj/mixxx/pull/4531) [lp1889352](https://bugs.launchpad.net/mixxx/+bug/1889352)
```

**Review by ronso0 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**